Content.Shared/AirlockPainter/Components/PaintableAirlockComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.AirlockPainter
+{
+    [RegisterComponent]
+    public sealed class PaintableAirlockComponent : Component
+    {
+        [DataField("group")]
```

Comment:
Adding a custom type serializer will allow the yaml linter to check for errors, by making sure that each string actually corresponds to a valid prototype
```suggestion
        [DataField("group", customTypeSerializer:typeof(PrototypeIdSerializer<AirlockGroupPrototype>))]
```